### ISSUE TITLE
rng2 PTN001020756 - PTHR14149

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#3057]
please remove from

O14188

GO:0000142 | cellular bud neck contractile ring | IBA with Q5AH02 , S000006163 , PTN001020756

pombe does not have a cellular bud neck contractile ring (I'm not sure that such a term should exist but since it does...)

**Comment by pgaudet** [comment]
Only cerevisiae has a cellular bud neck, right ? Shouldn't we have a taxon constraint ?

**Comment by pgaudet** [comment]
Fixed

**Comment by ValWood** [comment]
https://www.ebi.ac.uk/QuickGO/term/GO:0005935
cellular bud neck

has
GO:0005933 | cellular bud | Never in Taxon | 4895 | Schizosaccharomyces

Is that working?

**Comment by pgaudet** [comment]
I guess - but is this biologically correct ? All other yeasts bud ?

**Comment by ValWood** [comment]
Possibly.

No, but I don't know which yeast have budding  and which don't.....
I will ask on the Steve's group call today for a good phylogenetic restriction.

I guess all non-fission yeast bud in yeast growth stage (i.e fission vs budding), would make sense, but I'm not completely sure.

@mah11 do you know?

Possibly "never in Schizosaccharomyces" is the best we can do.

**Comment by mah11** [comment]
Many yeasts other than Saccharomyces reproduce by budding, and therefore have bud necks. I don't know enough to say that it's *all* yeasts other than Schizosaccharomyces, but I strongly suspect that it's at least most of them.

I have just found that yeasts are extremely taxonomically diverse: there are fungi with yeast forms in both ascomycetes and basidiomycetes. That means an "only in" constraint couldn't be more specific than Dikarya (and that wouldn't avoid the need for "never in Schizosaccharomyces" anyway).

tl;dr: yeah, just stick with "never in Schizosaccharomyces"

**Comment by pgaudet** [comment]
Thanks !
